backend/compact-connect/lambdas/cloudfront-csp/README.md

```diff
@@ -0,0 +1,46 @@
+# Cloudfront CSP Header Edge Lambda
```

Comment:
I admittedly don't know AWS well enough, but it seems odd to me that setting CSP headers is in its own lambda. Is that just "the way"?

Reply:
Good question. This is technically running in the context of a CDN viewer response. I updated this to reflect that.